src/Admin/Pool.php

```diff
@@ -238,13 +238,13 @@ public function hasAdminByClass($class)
     public function getAdminByAdminCode($adminCode)
     {
         $codes = explode('|', $adminCode);
+        $code = array_shift($codes);
 
-        if (false === $codes) {
+        if ('' === $code) {
             return false;
         }
 
-        $admin = $this->getInstance($codes[0]);
-        array_shift($codes);
+        $admin = $this->getInstance($code);
 
         if (empty($codes)) {
```

Comment:
Do we still need this check?

Reply:
Not really, if `$codes` is an empty array, `$admin` will be left intact below.
Good catch! Thank you.